### PR TITLE
PREAPPS-6138: Handle attachments when ci / cd keys are not present in MIME

### PR DESCRIPTION
**Pull request by madhavdhuppe**
Set the contentDisposition in the attachment item as per their classification while normalizing mime parts.

### CODE REVIEW

File: src/utils/normalize-mime-parts.ts [hunk 1]
```diff
@@ -77,9 +77,19 @@ export function normalizeMimeParts(
...
 ) {
-	const processAttachment = ({ ...attachment }) => {
+	const processAttachment = (
+		{ ...attachment },
+		forcedContentDisposition?: any
+	) => {
 		attachment.messageId = attachment.messageId || message.id;
 		attachment.url = getAttachmentUrl(attachment, { origin, jwtToken });
+
+		if (forcedContentDisposition === 'attachments') {
```
Comment (silentsakky) [inline src/utils/normalize-mime-parts.ts:87]: this should be added only when original attachment data doesn't have content-disposition header
we shouldn't change value if already present

File: src/utils/normalize-mime-parts.ts [hunk 2]
```diff
@@ -77,9 +77,19 @@ export function normalizeMimeParts(
 		isDesktop
 	}: { isDesktop?: string; jwtToken?: string; origin?: string }
 ) {
-	const processAttachment = ({ ...attachment }) => {
+	const processAttachment = (
+		{ ...attachment },
+		forcedContentDisposition?: any
```
Comment (silentsakky) [inline src/utils/normalize-mime-parts.ts:82]: type should be string instead of any

File: src/utils/normalize-mime-parts.ts (hunk 1 above)
Comment (madhavdhuppe) [reply src/utils/normalize-mime-parts.ts:87]: Actually we are doing the same now as well but will add conditional code for betterment

File: src/utils/normalize-mime-parts.ts (hunk 2 above)
Comment (madhavdhuppe) [reply src/utils/normalize-mime-parts.ts:82]: sure

**Review by silentsakky — APPROVED** [review]

**Review by BhavinBathani — APPROVED** [review]

**Pull request merged**